app/components/shared/inputs/NumberInput.vue.ts

```diff
@@ -1,16 +1,27 @@
 import { Component, Prop } from 'vue-property-decorator';
 import { BaseInput } from './BaseInput';
 import { INumberMetadata } from './index';
+import { String } from '../../../../node_modules/aws-sdk/clients/ssm';
```

Comment:
?

Reply:
fixed! sorry was a typo that vscode assumed was something else :(

Reply:
haha automatic import in VSCode sometimes produces weird results lol